llvm/tools/llvm-foreach/llvm-foreach.cpp

```diff
@@ -62,6 +62,13 @@ static cl::opt<std::string> OutputFileList{
     "out-file-list", cl::desc("Specify filename for list of outputs."),
     cl::value_desc("filename"), cl::init("")};
 
+static cl::opt<std::string> OutIncrement{
```

Comment:
Can we add a test for this new option?
The example of llvm-foreach tests can be found in `llvm/llvm/test/tools/llvm-foreach` folder.

Reply:
@Fznamznon, I augmented the existing tests to use the new option, would you like completely separate tests?

Reply:
@Fznamznon, @AlexeySachkov - ping

Reply:
Oh, I'm very sorry. I somehow missed both test (probably because tests are usually a the end of review) and ping. That works for me, thanks!